src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,14 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] balls = new Ball[3];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = lottery.getRandomBall();
+        }
+
+        for (int i = 0; i < balls.length; i++) {
+            balls[i].toString();
```

Comment:
The method toString() is called here, but its result is not used. If you intended to print the ball information to the console, you should use System.out.println(balls[i].toString()). Moreover, since println() internally calls the toString() method of its argument, you can simplify it to System.out.println(balls[i]).